relations/controllers.py

```diff
@@ -31,6 +30,32 @@ def service_status(params: MultiDict) -> Response:
     return {'iam': 'ok'}, HTTPStatus.OK, {}
 
 
+def relation2dict(rel: Relation) -> Dict[str, Any]:
```

Comment:
This is totally fine; another approach might be to use [``_asdict()``](https://docs.python.org/3/library/collections.html#collections.somenamedtuple._asdict).